boards/arm/96b_carbon/doc/96b_carbon.rst

```diff
@@ -10,6 +10,39 @@ Zephyr applications use the 96b_carbon configuration to run on the 96Boards
 Carbon hardware. It is based on the STMicroelectronics STM32F401RET Cortex-M4
 CPU and also contains a nRF51832 chip connected over SPI for BLE connectivity.
 
+.. note::
+
+   The nRF51832 chip on this board is supported by Zephyr using
+   another configuration, 96b_carbon_nrf51; see
+   :ref:`96b_carbon_nrf51_board` for more details on this
+   configuration.
+
+   **Although both chips exist on the same physical board,
+   96b_carbon_nrf51 is a distinct Zephyr configuration, targeting a
+   different chip, than the 96b_carbon configuration described on this
+   page**.
+
+   This can be confusing. The important things to understand are:
```

Comment:
The situation which is confusing is that two things which Zephyr calls a "BOARD" are present on the same physical printed circuit board. I think this confusion is due to the terminology adopted by Zephyr, which usually makes sense but breaks down in edge cases like this. If @dbkinder also objects I will remove this, but I prefer to let it stand, as I think the confusion is being explained by this documentation rather than caused by it.